.ci/validation.sh

```diff
@@ -201,7 +201,7 @@ no-error-xwiki)
   cd ..
   checkout_from https://github.com/xwiki/xwiki-platform.git
   cd .ci-temp/xwiki-platform
-  git checkout "b31a47e4f84b1981c27fd55716c""eb89f6983af5b"
+  git checkout "e97a2a92a130061d21606a4310fe84b201a3d95b"
```

Comment:
if regression was found in https://github.com/xwiki/xwiki-platform/pull/1652/files

kind the same upate in Inputs should be on our side also to prove that we work on enums.
It will make us rely on thirparty project code less and make us detect regression by UTs executions.
please extend some Input with enum wihtout docs.

Reply:
done